x-pack/plugins/rule_registry/server/utils/create_lifecycle_executor.ts

```diff
@@ -293,10 +308,10 @@ export const createLifecycleExecutor =
       allEventsToIndex
         .filter(({ event }) => event[ALERT_STATUS] !== ALERT_STATUS_RECOVERED)
         .map(({ event }) => {
-          const alertId = event[ALERT_INSTANCE_ID]!;
+          const alertInstanceId = event[ALERT_INSTANCE_ID]!;
           const alertUuid = event[ALERT_UUID]!;
           const started = new Date(event[ALERT_START]!).toISOString();
-          return [alertId, { alertId, alertUuid, started }];
+          return [alertInstanceId, { alertId: alertInstanceId, alertUuid, started }]; // Todo: change object key from alertId to instanceId.
```

Comment:
nit: We can remove comment as it seems to be updated to `instanceId` now.

Reply:
@benakansara Not sure I follow. It is not updated to `instanceId` at the moment. 

It was left unchanged because this variable is being consumed by (at least) one additional function outside of the executor (line 213, `const result = await fetchExistingAlerts(`). I wanted to keep the scope to this as to not blow up the PR. Hence the `Todo`.

Reply:
ah, do you mean in here - 
```
const trackedAlertStateRt = rt.type({
  alertId: rt.string,
  alertUuid: rt.string,
  started: rt.string,
});
```

I was referring to this change - return [**alertId**, { alertId, alertUuid, started }]; => return [**alertInstanceId**, { alertId: alertInstanceId, alertUuid, started }]

In that case, makes sense, we can do this in another PR.